pkg/batch/scan_test.go

```diff
@@ -428,3 +431,43 @@ func BenchmarkScan(b *testing.B) {
 		}
 	}
 }
+
+var letterRunes = []rune("abcdefghijklmnopqrstuvwxyzABCDEFGHIJKLMNOPQRSTUVWXYZ,")
+
+func RandStringRunes(n int) string {
```

Comment:
```suggestion
func RandString(n int) string {
```
You're returning string not runes

Reply:
Indeed